app/views/availabilities/places/edit.html.erb

```diff
@@ -0,0 +1,72 @@
+<div class="container-fluid places-container">
+  <%= form_tag(availabilities_place_path, method: "put", class: "edit-places-form row") do %>
+    <div class="col-md-3 edit-places-nav">
+      <div style="display: flex; height: 93px; padding: 8px 0; justify-content: center; align-items: center; border-bottom: 1px solid #e0e0e0">
+        <h1 style="flex: none;"><%= @place.name %></h1>
+      </div>
+      <div style="display: flex; flex-direction: column; padding: 32px 40px; justify-content: space-around;">
+        <%= submit_tag("Update", id: 'update-button', data: { disable_with: false }, class: "btn btn-large btn-primary btn-nav") %>
+        <%= submit_tag("Cancel", id: 'cancel-button', data: { disable_with: false }, class: "btn btn-large btn-danger btn-nav") %>
+        <%= link_to "Back", availabilities_places_path, class: "btn btn-large btn-light btn-nav" %>
+      </div>
+    </div>
+
+    <div class="col-md-9" style="height: 100%;">
+      <div class="edit-places-table">
+        <div class="edit-places-heading">
+          <div class="edit-places-empty-heading">
+          </div>
+          <div class="heading-group">
+            <% (1..7).each do |index| %>
+              <div class="places-day-heading">
+                <div class="places-day-text">
+                  <%= Availability.days.keys[index % 7][0..2] %>
+                </div>
+              </div>
+            <% end%>
+            <div style="overflow: scroll;"></div>
+          </div>
+        </div>
+        <div id="scroll-table">
+          <div class="time-heading">
+            <% (0..23).each do |x| %>
+              <div class="time-block">
+                <span class="time-text"<% if x == 0 %>, style="display: none;"<% end %>>
+                  <%= "%02d:00" % x %>
+                </span>
+              </div>
+            <% end %>
+          </div>
+          <div class="table-cells">
+            <div>
+              <% (0..23).each do |x| %>
+                <div class="row-lines"></div>
+              <% end %>
+            </div>
+            <% (1..7).each do |day| %>
+              <div class="table-columns">
+                <div class="table-columns-padded">
+                  <% @timeslots[day % 7].each do |ts| %>
+                    <% tr = ts.time_range %>
+                    <div class="timeslot" style="
+                      top: <%= calc_offset(tr)%>px;
+                      height: <%= calc_slot_height(tr) %>px;">
+                      <div class="timeslot-vertical"></div>
+                      <%= generate_select(ts) %>
+                    </div>
```

Comment:
I mean you have 11 closing tags here lol. Breaking them into partials / helpers would make it much easier to read

Reply:
Done